### PR TITLE
Use `homeassistant/` as discovery prefix in mqtt examples

### LINKED ISSUE #27550: MQTT: Is it the default topic "home-assistant" or "homeassistant"? The doku uses both.
### Feedback

The examples in the doku use both topics. So what is the default auto discovery topic?

### URL

https://www.home-assistant.io/integrations/mqtt/

### Version

2023.5.4

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by jbouwh**
## Proposed change

Use `homeassistant/` as discovery prefix in mqtt examples.
Ad missing closing bracket.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #27550

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/mqtt.markdown [hunk 1]
```diff
@@ -109,7 +109,7 @@ With a secure broker connection it is possible to use a client certificate for a
 
 #### Using WebSockets as transport
 
-You can select `websockets` as transport method if your MQTT broker supports it. When you select `websockets` and click `NEXT` you will be able to add a WebSockets path (default = `/` and WebSockets headers (optional). The target WebSockets URI: `ws://{broker}:{port}{WebSockets path}` is built with `broker`, `port` and `ws_path` (WebSocket path) settings.
+You can select `websockets` as transport method if your MQTT broker supports it. When you select `websockets` and click `NEXT` you will be able to add a WebSockets path (default = `/`) and WebSockets headers (optional). The target WebSockets URI: `ws://{broker}:{port}{WebSockets path}` is built with `broker`, `port` and `ws_path` (WebSocket path) settings.
```
Comment (c0ffeeca7) [inline source/_integrations/mqtt.markdown]:
```suggestion
You can select `websockets` as transport method if your MQTT broker supports it. When you select `websockets` and click `NEXT`, you will be able to add a WebSockets path (default = `/`) and WebSockets headers (optional). The target WebSockets URI: `ws://{broker}:{port}{WebSockets path}` is built with `broker`, `port` and `ws_path` (WebSocket path) settings.
```

File: source/_integrations/mqtt.markdown [hunk 2]
```diff
@@ -218,6 +218,7 @@ The discovery topic needs to follow a specific format:
 <discovery_prefix>/<component>/[<node_id>/]<object_id>/config
 ```
 
+- `<discovery_prefix>`: The Discovery Prefix, defaults to `homeassistant`. This prefix be [changed](#discovery-options).
```
Comment (c0ffeeca7) [inline source/_integrations/mqtt.markdown]:
```suggestion
- `<discovery_prefix>`: The Discovery Prefix defaults to `homeassistant`. This prefix be [changed](#discovery-options).
```

**Review by c0ffeeca7 — DISMISSED** [review]
Tiny tweak

File: source/_integrations/mqtt.markdown (hunk 2 above)
Comment (jbouwh) [inline source/_integrations/mqtt.markdown]:
```suggestion
- `<discovery_prefix>`: The Discovery Prefix defaults to `homeassistant`. This prefix can be [changed](#discovery-options).
```

**Review by frenck — APPROVED** [review]

**Pull request merged**